src/app/components/FormInput.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Input from 'react-toolbox/lib/input';
+
+
+const FormInput = ({
```

Comment:
@kudzayimutunga Really need to work on your indentation - not the tab spacing in question here